File: Basics/Conditional-Statements-Advanced/Lab/ski_trip.py
stay_days = int(input())
room_kind = input()
appraisal = input()
nights = stay_days - 1
discount = 0
end_sum = 0

if room_kind == "room for one person":
    room_price = 18
    end_sum = room_price * nights
elif room_kind == "apartment":
    room_price = 25
    end_sum = room_price * nights
    if stay_days < 10:
        discount = end_sum * 0.30
    elif stay_days <= 15:
        discount = end_sum * 0.35
    elif stay_days > 15:
        discount = end_sum * 0.50
elif room_kind == "president apartment":
    room_price = 35
    end_sum = room_price * nights
    if stay_days < 10:
        discount = end_sum * 0.10
    elif stay_days <= 15:
        discount = end_sum * 0.15
    elif stay_days > 15:
        discount = end_sum * 0.20

end_sum -= discount

if appraisal == "positive":
    end_sum += end_sum * 0.25
else:
    end_sum -= end_sum * 0.10

print(f"{end_sum:.2f}")

File: JS-Basics/0-3-Conditional-Steatements-Advanced/Lab/ski-trip.js
function skiTrip(input) {
    let nightsCount = Number(input[0]) - 1;
    let roomType = input[1];
    let appraisal = input[2];
    let price;
    if (roomType == "room for one person") {
        price = 18 * nightsCount;
    } else if (roomType == "apartment") {
        price = 25 * nightsCount;
        if (nightsCount < 10) {
            price -= price * 0.30;
        }else if (nightsCount <= 15){
            price -= price * 0.35; 
        } else if (nightsCount > 15) {
            price -= price * 0.50;
        }
    } else if (roomType == "president apartment") {
        price = 35 * nightsCount;
        if (nightsCount < 10) {
            price -= price * 0.10;
        }else if (nightsCount <= 15){
            price -= price * 0.15; 
        } else if (nightsCount > 15) {
            price -= price * 0.20;
        }
    }

    if (appraisal == "positive") {
        price += price * 0.25;
    } else {
        price -= price * 0.10;
    }

    console.log(price.toFixed(2));
}

skiTrip(["14", "apartment", "positive"])
skiTrip(["30", "president apartment", "negative"])
